### DISCUSSION TITLE
Turn Off Auto Collapse

### DISCUSSION THREAD

**Discussion by 9812ga98sd** [TriliumNext/Trilium#1751] [Q&A]
I used trilium every day. Its great. However I need to turn off the auto collapse because I have set notes open in multiple places. Also using tabs doesn't work as I use the list with 30 open/close areas resulting in 30 tabs...

Can it be toggled?

**Comment by zadam** [comment]
This is not possible to turn off at the moment

**Reply by 9812ga98sd** [reply]
Is there a way to change the time it auto collapses? For myself i have to keep reopening the same subtrees and its difficult to rememeber which ones i had open. Or the ability to pin trees to be always open?

**Comment by zadam** [comment]
I added a settings to disable this to a stable branch, will be in the next release:

![image](https://user-images.githubusercontent.com/617641/111683871-502a0d00-8826-11eb-8bd0-49ccaa15f318.png)

**Reply by 9812ga98sd** [reply]
Great thats Perfect! I have moved over all my notes from other programs. Thanks

**Reply by 9812ga98sd** [reply]
Is this in the new trilium-windows-x64-0.46.6 release? I can't find a settings button. 

EDIT: its in the bottom of the main left hand window.
